spring-boot-starter/spring-boot-starter-registry/pom.xml

```diff
@@ -0,0 +1,42 @@
+<!--
+  ~ Copyright 2017 Huawei Technologies Co., Ltd
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~    http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<parent>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<groupId>io.servicecomb</groupId>
+		<version>0.6.0-SNAPSHOT</version>
+	</parent>
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>spring-boot-starter-registry</artifactId>
+
+	<dependencies>
+		<dependency>
+			<groupId>io.servicecomb</groupId>
+			<artifactId>service-registry</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>io.servicecomb</groupId>
+			<artifactId>handler-loadbalance</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-web</artifactId>
```

Comment:
Do we need to have this dependency?

Reply:
Yes. To extend RestTemplate we use httpclient apis from this.

Reply:
we can remove dependency to handler-loadbalance and spring-web and not need to extend RestTemplate

Reply:
deleted RestTemplate